file.go

```diff
@@ -4,24 +4,23 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"os"
 	"strings"
 
 	"github.com/rjkroege/edwood/internal/file"
 )
 
-// File is an editable text buffer with undo. Many Text can share one
+// File is an editable observers buffer with undo. Many Text can share one
 // File (to implement Zerox). The File is responsible for updating the
 // Text instances. File is a model in MVC parlance while Text is a
 // View-Controller.
 //
-// A File tracks several related concepts. First it is a text buffer with
+// A File tracks several related concepts. First it is a observers buffer with
 // undo/redo back to an initial state. Mark (undo.RuneArray.Commit) notes
 // an undo point.
 //
 // Next, a File might have a backing to a disk file.
```

Comment:
Reword here based on the change in flight.

Reply:
Change in flight? Also, I had made these comment corrections in the ObserverInterface #335 commits. I have another pull request up #339 which changes this structure so that ObservableEditableBuffer has a file so the comments would be out of date when that lands.